localstack-core/localstack/openapi.yaml

```diff
@@ -568,6 +600,8 @@ paths:
   /_localstack/diagnose:
     get:
       description: Get diagnostics report
+      operationId: get_diagnose
```

Comment:
`get` and `diagnose` are both verbs.

maybe `get_diagnosis` or `get_diagnostics`?